nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -246,14 +266,15 @@ BtNavigator::navigateToPose()
       return action_server_->is_cancel_requested();
     };
 
-  auto bt_xml_filename = action_server_->get_current_goal()->behavior_tree;
+  std::string bt_xml_filename = action_server_->get_current_goal()->behavior_tree.c_str();
 
   // Empty id in request is default for backward compatibility
   bt_xml_filename = bt_xml_filename.empty() ? default_bt_xml_filename_ : bt_xml_filename;
+  RCLCPP_DEBUG(get_logger(), "Check if we need to load BT: %s", bt_xml_filename.c_str());
 
   if (!loadBehaviorTree(bt_xml_filename)) {
     RCLCPP_ERROR(
-      get_logger(), "BT file not found: %s. Navigation canceled",
+      get_logger(), "BT file not found: %s. Navigation canceled. Current bt: %s",
```

Comment:
Remove `current BT:` might be useful for debugging, but not relevant in the log for what this is trying to communicate

Reply:
ah.. yes. because you only care about the new file/bt and not what was once in the past. I'll delete it!